embc-app/ClientApp/src/app/app.module.ts

```diff
@@ -128,6 +129,7 @@ import { TopTabBarComponent } from './components/top-tab-bar/top-tab-bar.compone
     ReferralListItemComponent,
     AbstractReferralComponent,
     TopTabBarComponent,
+    AdminVolunteerMakerComponent,
```

Comment:
Could you please move this up beside the other Provincial Admin pages?
And actually, it would be nice if they were named consistently :)

Reply:
Done in other branch